### PR TITLE
small fix to russian revolver sounds to stop accidental cheaters

### PR DESCRIPTION
**Pull request by MRLinew**
## About the PR 

moves the click sound to where the gun is actually being used. Before it would click whenever you clicked, even if whoever was using it didnt actually try to shoot the gun.

## Why's this needed? 
Lots of people were unknowingly cheating at russian roullete, by clicking anywhere on the screen and assuming the click sound meant it worked. Now the click sound only plays when you have an attempt.

## Changelog

```changelog
(u)Froggit_Dogget
(+)Fixed the russian revolver sounds to stop people from thinking they used it when they didn't.
```

### CODE REVIEW

**Comment by MRLinew** [comment]
i think i did this right; first time

**Comment by ZeWaka** [comment]
> i think i did this right; first time

I would've named the PR properly, and you should learn how to apply your own labels eventually, but looks good 👍

File: code/obj/item/gun/russian.dm [hunk 1]
```diff
@@ -35,9 +35,9 @@
 	proc/fire_gun(mob/user as mob)
 		if(src.shotsLeft > 1)
 			src.shotsLeft--
-			playsound(user, "sound/weapons/Gunclick.ogg", 80, 1)
 			for(var/mob/O in AIviewers(user, null))
 				if (O.client)
+					playsound(user, "sound/weapons/Gunclick.ogg", 80, 1)
 					O.show_message("<span class='alert'>[user] points the gun at \his head. Click!</span>", 1, "<span class='alert'>Click!</span>", 2)
```
Comment (ZeWaka) [inline code/obj/item/gun/russian.dm:41]: I'm not sure this is the right fix. This'll play the sound for every single viewer. so if you have 50 people viewing you, it'll play the sound 50 times....

this whole thing should just be src.visible_message

**Review by ZeWaka — CHANGES_REQUESTED** [review]

File: code/obj/item/gun/russian.dm (hunk 1 above)
Comment (MRLinew) [reply code/obj/item/gun/russian.dm:41]: oh

**Pull request closed without merging**